Jenkinsfile

```diff
@@ -39,7 +39,7 @@ node {
     }
   }
 
-  def oppossumCI = docker.image('hyrise/opossum-ci:18.04');
+  def oppossumCI = docker.image('hyrise/opossum-ci:18.10');
```

Comment:
Do we have no image for `19.04`?

Reply:
Mmmh, no idea what I did there. That was... the entire point of this PR. oO

Hopefully this also solves the coverage issue, as 19.04 has a newer clang version.